.github/workflows/go.yml

```diff
@@ -5,10 +5,10 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
-    - name: Set up Go 1.13
-      uses: actions/setup-go@v1
```

Comment:
why not use a matrix to test minimum required go version and latest?

Reply:
The latest commit now uses matrix to test on Go 1.16 and Go 1.17